pnet_transport/Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["networking", "transport", "raw", "socket"]
 categories = ["network-programming"]
 
 [dependencies]
-libc = "0.2"
-pnet_base = { path = "../pnet_base", version = "0.20.0" }
-pnet_sys = { path = "../pnet_sys", version = "0.20.0" }
+libc        = ">=0.2.34"
+pnet_base   = { path = "../pnet_base", version = "0.20.0" }
+pnet_sys    = { path = "../pnet_sys", version = "0.20.1" }
```

Comment:
Please don't align-format like this. One space before the `=` and one after it is perfectly fine.

The problem with align formatting is that if we introduce a new dependency with a new name then all these lines must be changed, instead of just adding one single line. That results in more work just to update formatting and larger diffs than needed as well as broken `git blame`.

Reply:
okay